### ISSUE TITLE
Cannot attach image

### ISSUE THREAD

**Issue by susobaco** [LubuWest/Friendiqa#9]
Hello, in version 2.1 I tried to send a photo in a comment and it was impossible to attach it. This also happens to me in version 2.2. I'm on android 4.4. How to send a log? in alogcat there is no activity

**Comment by LubuWest** [comment]
The attachment button is not working  in all circumstances. I have  a look...

**Comment by LubuWest** [comment]
Can you check if the error is fixed with version  0.3?

**Comment by susobaco** [comment]
No, the problem has not been solved. Do you know any adb commands to get a log with debug?

**Comment by LubuWest** [comment]
ADB logcat gives the same output as  the logcat apps. The photo attachment works for me. Could you provide more information? Do you see  the image in the messageSend window? Do you have  permissions set. Are you sending  the message successful but the image dies not appear? Any error messages?

**Comment by susobaco** [comment]
Hi, I'm answering the questions:
I don't see the image in the messageSend window. The permissions I think are correct, and the message is sent but no image is published.
Attached logcat
[logcat.txt](https://github.com/LubuWest/Friendiqa/files/2179093/logcat.txt)

**Comment by LubuWest** [comment]
The logcat does  not show any output from Friendiqa. Did you select an image from  the gallery via the image attachment button?

**Comment by susobaco** [comment]
Yes, I add the image from the button, it can be done in another way?. I will try to uninstall the application and reinstall it to see what happens.

**Comment by LubuWest** [comment]
Yes, you can send one or  more  images directly from the gallery via share function. However, this obviously doesn't work for comments.

**Comment by susobaco** [comment]
Hey, sorry about the wait. Yes, I can share images from the gallery, but not with the add button.

**Comment by LubuWest** [comment]
I just pushed a new version to fdroid repository, this should work now!
